### PR TITLE
TestEngine: Migrate current Framework's ut

### PR DESCRIPTION
**Pull request by shargon**
Better to merge https://github.com/neo-project/neo-devpack-dotnet/pull/969 before
Part of https://github.com/neo-project/neo-devpack-dotnet/issues/949

This pr migrate the unit tests from `SmartContract.Framework` to the new `TestEngine`.

It allow to check the coverage of the tested contracts in Coveralls, easier to use and easier to test.

Note: Most of the changed files are artifacts autogenerated from the tested nefFiles.

### CODE REVIEW

**Comment by shargon** [comment]
Only left `BlockchainTest` 😪

**Comment by shargon** [comment]
Finally, one of the two projects from the old test engine has been migrated :)

**Comment by shargon** [comment]
Wait for one fix that I want to do before merge.

File: src/Neo.SmartContract.Testing/Native/LedgerContract.cs [hunk 1]
```diff
@@ -34,19 +33,19 @@ public abstract class LedgerContract : SmartContract
...
     [DisplayName("getBlock")]
-    public abstract TrimmedBlock? GetBlock(byte[]? indexOrHash);
+    public abstract Models.Block? GetBlock(byte[]? indexOrHash);
 
     /// <summary>
     /// Safe method
     /// </summary>
     [DisplayName("getTransaction")]
-    public abstract Transaction? GetTransaction(UInt256? hash);
+    public abstract Models.Block? GetTransaction(UInt256? hash);
```
Comment (shargon) [inline src/Neo.SmartContract.Testing/Native/LedgerContract.cs:42]:
```suggestion
    public abstract Models.Transaction? GetTransaction(UInt256? hash);
```

**Comment by shargon** [comment]
Ready to review

**Review by Jim8y — APPROVED** [review]

**Pull request merged**